src/mock/network.py

```diff
@@ -7,12 +7,19 @@
 
 class objectracker(object):
     """Mock object tracker"""
+
     missingObjects = {}
 
 
 class stats(object):
     """Mock network statics"""
+
     @staticmethod
     def connectedHostsList():
-        """List of all the connected hosts"""
-        return ()
+        """List of all the mock connected hosts"""
+        return [
+            "141.95.47.146:8448",
```

Comment:
use strings or ints as this is not the correct syntax, and people may be confused thinking this is correct syntax. If we use instead, say, "conn1", "conn2", or "a", "b", "c", this is less confusing